static/app/components/searchQueryBuilder/useQueryBuilderState.tsx

```diff
@@ -101,6 +114,70 @@ function updateFreeText(
   };
 }
 
+function updateFilterMultipleValues(
+  state: QueryBuilderState,
+  token: TokenResult<Token.FILTER>,
+  values: string[]
+) {
+  const uniqNonEmptyValues = Array.from(
+    new Set(values.filter(value => value.length > 0))
+  );
+  if (uniqNonEmptyValues.length === 0) {
+    return {...state, query: replaceQueryToken(state.query, token.value, '')};
+  }
+
+  const newValue =
+    uniqNonEmptyValues.length > 1
+      ? `[${uniqNonEmptyValues.join(',')}]`
+      : uniqNonEmptyValues[0];
+
+  return {...state, query: replaceQueryToken(state.query, token.value, newValue)};
+}
+
+function mutliSelectTokenValue(
```

Comment:
```suggestion
function multiSelectTokenValue(
```
last one I promise
(One usage of this function below too)

Reply:
How do I keep messing this up hah